### PR TITLE
Add mojopaste (pastebin)

### PR DESCRIPTION
**Pull request by Corion**
Thank you for taking the time to work on a PR for Awesome-selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -714,6 +714,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [gist](https://gitlab.deblan.org/deblan/gist) - GIST is an open-source application to share code. `GPLv3` `PHP`
   * [hastebin](http://hastebin.com/about.md) - Open source pastebin written in node.js. ([Demo](http://hastebin.com/), [Source Code](https://github.com/seejohnrun/haste-server)) `MIT` `Nodejs`
   * [localpaste](https://github.com/petermaloney/localpaste) - Simple python based pastebin you can run locally, with curl for input. Supports RAW text by default and binary data such as images (no extensions on returned url). `GPLv2` `Python`
+  * [mojopaste])(http://search.cpan.org/dist/App-mojopaste/) - Perl based pastebin - `Perl` `ARTv2`
   * [NoteHub](https://notehub.org) - Free and Hassle-free Pastebin for Markdown Pages. Simple, clean, password provided, generated-short link. ([Demo](https://notehub.org), [Source Code](https://github.com/chmllr/NoteHub)) `MIT` `Nodejs`
```
Comment (n8225) [inline README.md]: Remove the - after the description and a full stop.

**Comment by n8225** [comment]
Source code: [https://github.com/jhthorsen/app-mojopaste](https://github.com/jhthorsen/app-mojopaste)
Demo: [http://p.thorsen.pm/](http://p.thorsen.pm/)

File: README.md [hunk 2]
```diff
@@ -714,6 +714,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [gist](https://gitlab.deblan.org/deblan/gist) - GIST is an open-source application to share code. `GPLv3` `PHP`
   * [hastebin](http://hastebin.com/about.md) - Open source pastebin written in node.js. ([Demo](http://hastebin.com/), [Source Code](https://github.com/seejohnrun/haste-server)) `MIT` `Nodejs`
   * [localpaste](https://github.com/petermaloney/localpaste) - Simple python based pastebin you can run locally, with curl for input. Supports RAW text by default and binary data such as images (no extensions on returned url). `GPLv2` `Python`
+  * [mojopaste](http://search.cpan.org/dist/App-mojopaste/) - Perl based pastebin. ([Demo](http://p.thorsen.pm/) )[Source Code](https://github.com/jhthorsen/app-mojopaste) `Perl` `ARTv2`
```
Comment (n8225) [inline README.md]: ([Demo](http://p.thorsen.pm/) )[Source Code](https://github.com/jhthorsen/app-mojopaste)

Should be:
([Demo](http://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste))

**Comment by n8225** [comment]
That didn't come out right.  Damn mobile version.
See example above

File: README.md [hunk 3]
```diff
@@ -714,6 +714,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [gist](https://gitlab.deblan.org/deblan/gist) - GIST is an open-source application to share code. `GPLv3` `PHP`
   * [hastebin](http://hastebin.com/about.md) - Open source pastebin written in node.js. ([Demo](http://hastebin.com/), [Source Code](https://github.com/seejohnrun/haste-server)) `MIT` `Nodejs`
   * [localpaste](https://github.com/petermaloney/localpaste) - Simple python based pastebin you can run locally, with curl for input. Supports RAW text by default and binary data such as images (no extensions on returned url). `GPLv2` `Python`
+  * [mojopaste](http://search.cpan.org/dist/App-mojopaste/) - Perl based pastebin. ([Demo](http://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste)) `Perl` `ARTv2`
```
Comment (nodiscc) [inline README.md]: Please add an antry pointing to the Artistic License v2 in the Licenses List section

Comment (nodiscc) [inline README.md]: License goes before language: `ARTv2` `Perl`

**Comment by Kickball** [comment]
ARTv2 seems to be in the list of licenses on the master branch already.

**Pull request merged**